src/concepts/services/format.md

```diff
@@ -1,6 +1,41 @@
 # Format
 
-{% hint style="info" %}
-`Contribute to our docs:` [Explain the format module in depth](https://github.com/yewstack/docs/issues/24)
-{% endhint %}
+## Introduction
+Yew provides the format module to make it easy to convert from Rust types to common data formats (and vice versa). Rust provides support for a number of different data formats. Some of these formats are binary-only (e.g. CBOR) while others support both text and binary (e.g. JSON). 
 
+## Example usage
+```rust
+use yew::format::Json;
+let data = &"{\"key\": \"value\"}".to_string();
+// convert the data into JSON
+let json_data = Json(&data);
+// convert the JSON back
+let Json(dump) = json_data;
```

Comment:
nit: this example isn't actually doing any conversion. Could you change it to match what you have below with `Into`/`From` Text  or Binary

Reply:
@teymour-aldridge I don't think you addressed this comment